common/BlacklistedPriceRequests.js

```diff
@@ -0,0 +1,8 @@
+// Whitelisted price requests that will display on voter clients.
+const REQUEST_BLACKLIST = {
+  SOME_IDENTIFIER: ["1596666977"]
```

Comment:
this is just a dummy identifier, right?

Reply:
Yep just wanted to show the format. There is no harm in leaving this uncommented as `ActiveRequests` and `ResolvedRequests` only care if a pending or resolved request matches with something on this list